cypress/integration/DynamicTextInput.ts

```diff
@@ -0,0 +1,20 @@
+describe('DynamicTextInput', () => {
+  it('renders properly with Gold Light theme', () => {
+    cy.visitStory({ storyId: 'inputs-dynamictextinput--default', themeId: 'GoldLight' });
+
+    cy.get('[data-testid="input"]').clear();
+    cy.get('[data-testid="input"]').type('Some really really really long text...');
+
+    cy.percySnapshot('DynamicTextInput with Gold Light theme');
+  });
+
+  it('renders properly with Gold Dark theme', () => {
```

Comment:
We could get rid of this test case and add more Percy snapshots with different lenghts.